_data/gallery.yml

```diff
@@ -1,3 +1,92 @@
+- title:    "Be Consistent! Improving Procedural Text Comprehensionusing Label Consistency"
+  authors:  "Xinya Du, Bhavana Dalvi Mishra, Niket Tandon, Antoine Bosselut, Wen-tau Yih, Peter Clark, Claire Cardie"
+  venue:    "NAACL 2019"
+  links:
+  - paper:  "https://www.semanticscholar.org/paper/Be-Consistent-!-Improving-Procedural-Text-using-Du-Mishra/6a4b27f8bde3e8cd65f4d9633339f7149b878672"
+  - code:   "https://github.com/allenai/propara"
+
+###############################################################################
+- title:    "Structural Scaffolds for Citation Intent Classification in Scientific Publications"
+  authors:  "Arman Cohan, Waleed Ammar, Madeleine van Zuylen, Field Cady"
+  venue:    "ArXiv 2019"
```

Comment:
You have arxiv 2019 listed for all of the below, but some of them (maybe all, or most) were published at conferences.  We need to find the right venue for these, or it looks bad.

Reply:
@matt-gardner I figured they were prepublished in ArXiv and the conferences would be clear later.  I took this information from Semantic Scholar.  Do you think it should be easy to determine the conference now?